.github/workflows/docker-image.yml

```
@@ -0,0 +1,38 @@
+name: Push Docker Image
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-buildx-action@v2
+      - uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: generate image tag
+        id: generate-image-tag
+        run: |
+          IMAGE_NAME=yomiage-mon
+          IMAGE_ID=ghcr.io/${{ github.repository }}/$IMAGE_NAME
+
+          IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
```

`github.repository` は `approvers/yomiage-mon` になるのでこれで良さそう(元々名前全部lowercaseだから`tr`する意味はあんまり無さそうだけど)
```suggestion
          IMAGE_ID=$(echo 'ghcr.io/${{ github.repository }}' | tr '[A-Z]' '[a-z]')
```

---

https://github.com/approvers/yomiage-mon/pull/10/commits/8d6ae1302437e2748244053792fa0f4b644375c0 で対応